lib/linguist/generated.rb

```diff
@@ -113,6 +114,13 @@ def generated?
     def xcode_file?
       ['.nib', '.xcworkspacedata', '.xcuserstate'].include?(extname)
     end
+    
+    # Is the blob an IntelliJ IDEA file (IDE-specific files that are sometimes checked in)? 
```

Comment:
We might as well stay consistent with the other method descriptions:
```suggestion
    # Internal: Is the blob an IntelliJ IDEA project file?
    #
    # JetBrains IDEs generate project files under an `.idea` directory
    # that are sometimes checked into version control.
```

Reply:
Thanks. I do wonder, though, do you know the origin of the "Internal:" part? 😄 Is that a holdover from the days when Linguist was internal (maybe)?

Reply:
It's a holdover from Linguist's use of [TomDoc](http://tomdoc.org/#the-description-section) to generate API documentation:

> **Internal:** Indicates that the method is part of the project’s internal API. These are methods that are intended to be called from other classes within the project but not intended for public consumption. For example:
> ~~~ruby
> # Internal: Normalize the filename.
> ~~~